client/src/App.jsx

```diff
@@ -1,50 +1,76 @@
-const React = require('react');
-const ReactDOM = require('react-dom');
-const $ = require('jquery');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import FiveBooks from './components/FiveBooks.jsx';
 
 class App extends React.Component {
   constructor(props) {
-    super(props)
+    super(props);
 
     this.state = {
-      bookId: 0
-    }
+      authorInfo: {},
+    };
 
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
+    // this.handleChange = this.handleChange.bind(this);
+    // this.handleSubmit = this.handleSubmit.bind(this);
   }
 
-  handleChange (event) {
-    this.setState({bookId: event.target.value}, () => {
-      console.log('this.state.bookId', this.state.bookId);
-    })
-  }
+  // handleChange (event) {
+  //   this.setState({bookId: event.target.value}, () => {
+  //     console.log('this.state.bookId', this.state.bookId);
+  //   })
+  // }
 
-  handleSubmit (event) {
-    $.ajax({
-      url: 'author',
-      method: 'POST',
-      data: {bookId: this.state.bookId },
-      success: () => {
-        console.log('post worked');
-      }
-    })
+  // handleSubmit (event) {
+  //   $.ajax({
+  //     url: 'author',
+  //     method: 'POST',
+  //     data: { bookId: this.state.bookId },
+  //     success: () => {
+  //       console.log('post worked');
+  //     }
+  //   })
+
+  // }
 
+  componentDidMount() {
+    $.ajax({
+      url: '/author',
+      method: 'GET',
+      data: { bookId: 13 },
```

Comment:
I'm assuming this is placeholder, but don't forget to get this pulling from the URL eventually

Reply:
Yes, placeholder - I'll try to note 'placeholder' instances going forward so it's more clear.